docs/pages/cv/classification_reference.md

```diff
@@ -0,0 +1,297 @@
+*Example:* laboratory temperature, humidity, power fluctuations.
+
+#### CV example
+
+```obo
+relationship: part_of_workflow_stage MS:4000XXX ! chromatography stage
+```
+
+### 2. Information dependency type — `depends_on_data_type`
+
+**Definition:**
+Specifies which type of data input the metric requires to be computed.
+
+#### Subclasses
+
+* **Raw acquisition data:** metrics that can be calculated directly from the raw MS data, without identifications.
+  *Example:* total ion current stability, scan count.
+* **Deconvoluted data:** metrics based on processed spectra or peak lists obtained after signal deconvolution, centroiding, or deisotoping, but prior to identification.
+  *Example:* peak density in deconvoluted spectra, precursor mass range coverage.
+* **Identification results:** metrics that depend on identified peptides, compounds, or spectra.
+  *Example:* PSM-level FDR, peptide coverage.
+* **Quantification results:** metrics derived from quantitative data matrices.
+  *Example:* CV of peptide intensities.
+* **Hybrid:** metrics combining multiple data types (e.g., identification and quantification).
+* **Reference data:** metrics requiring comparison to external standards or reference files.
+  *Example:* RT deviation vs. iRT peptides, calibration QC.*
+
+#### CV example
+
+```obo
+relationship: depends_on_data_type MS:4000XXX ! raw acquisition data
+```
+
+### 3. Measurement scope — `has_measurement_scope`
+
+**Definition:**
+Indicates the level of data aggregation the metric summarizes.
+
+#### Subclasses
+
+* **Spectrum level:** per-spectrum metrics (e.g., number of peaks, S/N ratio).
+* **Pixel/voxel level:** per-pixel metrics in imaging or spatial omics.
+* **Feature level:** per feature (e.g., peptide, compound, or chromatographic peak).
+* **Run level:** aggregated per LC–MS run.
+* **Batch level:** aggregated across multiple related runs.
+* **Study level:** aggregated across an entire experiment or project.
+
+#### CV example
+
+```obo
+relationship: has_measurement_scope MS:4000XXX ! run level
+```
+
+### 4. Acquisition strategy — `applies_to_acquisition_mode`
+
+**Definition:**
+Specifies which acquisition mode or instrument configuration the metric is relevant for.
+
+#### Subclasses
+
+**Acquisition mode**
+
+* **Acquisition mode independent:** metrics valid for any acquisition method.
+* **Data-dependent acquisition (DDA):** metrics specific to stochastic precursor selection workflows.
+  *Example:* number of MS2 spectra per precursor.
+* **Data-independent acquisition (DIA):** metrics for window-based fragmentation strategies.
+  *Example:* precursor window purity.
+* **Targeted acquisition:** metrics for SRM, PRM, or other targeted workflows.
+  *Example:* transition reproducibility.
+* **Ion-mobility-coupled metric:** metrics derived from acquisition methods that include gas-phase ion mobility separation.
+  *Example:* TIMS mobility resolution (Δ1/K₀) per run.
```

Comment:
not sure if an explicit `Ion-mobility` value is needed/helpful, because it is orthogonal to DIA vs DDA,
and to be complete, we'd also need `LC coupled metric` (since this also adds another dimension, just as IM does) vs. direct injection.
We are mixing concepts here IMHO.
Aquisition mode (referring to the MassSpec itself) may just be
1) any
2) DDA
3) DIA
4) Targeted

other things like: LC, IM, Imaging are a different thing??
Even the specialized MSn is tricky, since TMT can be on MS3 level, yet still is DDA.

Now, one could just add multiple values for this dimension, but maybe we should push concepts like LC, IM etc into `workflow stage`? (where it would be duplicated anyways?)

Reply:
I agree that this mixes several orthogonal concepts: precursor selection logic (DDA/DIA/targeted), physical separation (LC, IM), and acquisition strategies (imaging, MSn)

Maybe a more coherent approach is to redefine this dimension strictly as "how ions are selected for fragmentation or measurement." In other words, capturing only the precursor selection logic of the mass spectrometer, independent of chromatography, IM, imaging, etc.

In that case, we can simplify the subclasses to:
- Acquisition-mode independent
- Data-dependent acquisition (DDA)
- Data-independent acquisition (DIA)
- Targeted acquisition